common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -472,7 +472,7 @@ def capture_stack(self, args, kwargs):
         """
         # pylint: disable=broad-except
 
-        stack = traceback.extract_stack()[:-2]
+        stack = [tuple(item) for item in traceback.extract_stack()[:-2]]
```

Comment:
type casting in tuple makes it compatible in both. As frame summary object tuple has same value as we got in py2